core/scripts/chaincli/handler/keeper_launch.go

```diff
@@ -274,182 +277,74 @@ func (k *Keeper) createKeeperJob(client cmd.HTTPClient, registryAddr, nodeAddr s
 	defer resp.Body.Close()
 
 	if resp.StatusCode >= 400 {
-		body, err := io.ReadAll(resp.Body)
+		body, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
 			return fmt.Errorf("failed to read error response body: %s", err)
 		}
 
 		return fmt.Errorf("unable to create keeper job: '%v' [%d]", string(body), resp.StatusCode)
 	}
-	log.Println("Keeper job has been successfully created in the Chainlink node with address: ", nodeAddr)
+
 	return nil
 }
 
-func (k *Keeper) launchChainlinkNode(ctx context.Context, port int) (string, func(), error) {
-	// Create docker client to launch nodes
-	dockerClient, err := client.NewClientWithOpts(client.WithAPIVersionNegotiation())
-	if err != nil {
-		return "", nil, fmt.Errorf("failed to create docker client from env: %s", err)
-	}
-
-	// Make sure everything works well
-	if _, err = dockerClient.Ping(ctx); err != nil {
-		return "", nil, fmt.Errorf("failed to ping docker server: %s", err)
-	}
-
-	// Pull DB image if needed
-	var out io.ReadCloser
-	if _, _, err = dockerClient.ImageInspectWithRaw(ctx, defaultPOSTGRESImage); err != nil {
-		log.Println("Pulling Postgres docker image...")
-		if out, err = dockerClient.ImagePull(ctx, defaultPOSTGRESImage, types.ImagePullOptions{}); err != nil {
-			return "", nil, fmt.Errorf("failed to pull Postgres image: %s", err)
-		}
-		out.Close()
-		log.Println("Postgres docker image successfully pulled!")
-	}
-
-	// Create DB container
-	dbContainerResp, err := dockerClient.ContainerCreate(ctx, &container.Config{
-		Image: defaultPOSTGRESImage,
-		Cmd:   []string{"postgres", "-c", `max_connections=1000`},
-		Env: []string{
-			"POSTGRES_USER=postgres",
-			"POSTGRES_PASSWORD=development_password",
-		},
-		ExposedPorts: nat.PortSet{"5432": struct{}{}},
-	}, nil, &network.NetworkingConfig{}, nil, "")
+const ocr2keeperJobTemplate = `type = "offchainreporting2"
+pluginType = "ocr2keeper"
+relay = "evm"
+name = "ocr2"
+schemaVersion = 1
+externalJobID = "%s"
+maxTaskDuration = "1s"
+contractID = "%s"
+ocrKeyBundleID = "%s"
+transmitterID = "%s"
+p2pv2Bootstrappers = [
+  "%s"
+]
+
+[relayConfig]
+chainID = %d
+
+[pluginConfig]
+maxQueryLength = 2000
+maxObservationLength = 2000
+maxReportLength = 2000`
```

Comment:
need to calculate comfortable upper bounds here

Reply:
it has been moved to the plugin repo